projects/packages/autoloader/src/class-version-selector.php

```diff
@@ -52,7 +52,10 @@ public function is_version_update_required( $selected_version, $compare_version
 	 * @return bool True if the version is a dev version, else false.
 	 */
 	public function is_dev_version( $version ) {
-		if ( 'dev-' === substr( $version, 0, 4 ) || '9999999-dev' === $version ) {
+		if ( 'dev-' === substr( $version, 0, 4 ) ||
+			'9999999-dev' === $version ||
+			str_ends_with( $version, '-alpha' ) ||
+			preg_match( '/-a(?:[.\d])+$/', $version ) ) {
```

Comment:
As you can see by the linter warning, can't use `str_ends_with()`. I'd just add it into the regex here.

Also I doubt we'll ever do like "a.1.1", so we can simplify it slightly

```suggestion
			preg_match( '/-(?:alpha|a\.\d+)$/', $version ) ) {
```